play/src/pusher/controllers/MemberController.ts

```diff
@@ -0,0 +1,68 @@
+import type { Request, Response } from "hyper-express";
+import { z } from "zod";
+import { authenticated } from "../middlewares/Authenticated";
+import { validateQuery } from "../services/QueryValidator";
+import { adminApi } from "../services/AdminApi";
+import { BaseHttpController } from "./BaseHttpController";
+
+/**
+ * Controller used for member related queries
+ * and mutations with admin api
+ */
+export class MemberController extends BaseHttpController {
+    routes(): void {
+        this.searchMembers();
+    }
+
+    /**
+     * @openapi
+     * /members:
+     *   get:
+     *     description: Search members from search term.
+     *     tags:
+     *      - Admin endpoint
+     *     parameters:
+     *      - name: "playUri"
+     *        in: "request"
+     *        required: true
+     *        type: "string"
+     *        description: The room url
+     *      - name: "searchText"
+     *        in: "request"
+     *        description: search text use for user search (name, email, etc)
+     *        required: true
+     *        type: "string"
+     *     responses:
+     *       200:
+     *         description: Member list or empty list.
+     *         example: "ok"
```

Comment:
TODO: describe the format of a member as a Zod type and reference the matching JSONSchema here.

Reply:
Fixed